rpc_jobs/standard_job_postmerge.yml

```diff
@@ -0,0 +1,141 @@
+- project:
+    name:       "gating-post-merge"
+    repo_url:   "https://github.com/rcbops/rpc-gating"
+    branch:     "master"
+    image:      "xenial"
+    scenario:   "functional"
+    action:     "test"
+    jobs:
+      - 'PM_{name}-{branch}-{image}-{scenario}-{action}'
+
+- job-template:
+    name: 'PM_{name}-{branch}-{image}-{scenario}-{action}'
+    project-type: pipeline
+    concurrent: true
+    FLAVOR: "performance1-1"
+    IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
+    properties:
+      - build-discarder:
+          num-to-keep: "30"
+      - github:
+          url: "{repo_url}"
+    parameters:
+      - rpc_gating_params
+      - instance_params:
+          IMAGE: "{IMAGE}"
+          FLAVOR: "{FLAVOR}"
+          REGIONS: "{REGIONS}"
+          FALLBACK_REGIONS: "{FALLBACK_REGIONS}"
+      - string:
+          name: REPO_URL
+          default: "{repo_url}"
+          description: Url of the repo under test
+      - string:
+          name: BRANCH
+          default: "{branch}"
+          description: Branch of the repo under test
+      - string:
+          name: STAGES
```

Comment:
Why are there four stages in `STAGES` but the bulk of the job is directly calling `stage`? I can see this is mostly copied from the pre-merge job template but any insight would be helpful.

Reply:
`STAGES` is for conditional stages, I can't see any reason why any of the hook stages would need to be skipped, so no point in making them conditional. The other instance related stages are in the shared library so the STAGES var must be set so they are not skipped. 
To show that I do not expect any stages to be skipped, I've have moved stages from a parameter to an env var that is set at the top of the DSL block.